vuework02/src/views/Step2.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div id="app">
+    <div class="worksheet">
+	<div class="title">
+      <span class="step">STEP2</span>
+      <p id="title">以下にお答えください</p>
+	</div>
+	<div>
+		<p>現在、生命保険に加入していますか？</p>
+		<input type="radio" value="Yes" @click="click">はい
+		<input type="radio" value="No" @click="click">いいえ
+	</div>
+	<div v-if="show">
+		<p>現在入院中ですか。または、最近3か月以内に医師の診断・検査の結果、入院・手術をすすめられたことはありますか？</p>
+		<input type="radio" value="Yes" @click="click2">はい
+		<input type="radio" value="No" @click="click2">いいえ
+	</div>
+	<div v-if="show2">
+		<p>過去5年以内に、病気やけがで、手術をうけたことまたは継続して7日以上の入院をしたことがありますか？</p>
+		<input type="radio" value="Yes" >はい
+		<input type="radio" value="No">いいえ
+	</div>
+    
+    </div>
+      <router-link tag="button" to="/">＜ 前に戻る</router-link> 
+      <router-link tag="button" to="/step3">次へ進む ＞</router-link> 
+	
+  </div>
+</template>
+
+<script>
+ export default{
+	data: function(){
+		return {
+			show:false,
+			show2:false
+		}
+	},
+	
+	methods:{
+		click:function(){
```

Comment:
①関数は動詞+名詞で命名します。clickという命名は抽象的で第三者がみたときにどういった処理を行っているのかわかりにくいので修正しましょう。click2という命名に関しても同様です。関数名に数字を用いることは一般的ではないですね。
②JS部では'{}'の両外側にはスペースを空けるのが一般的です。
③下記のようにも記述できますので覚えておきましょう。
```suggestion
		click() {
```